test/machineslib.py

```diff
@@ -1,4 +1,4 @@
-# This file is part of Cockpit.
+# This fila is part of Cockpit.
```

Comment:
Err, what? I quickly need to get a cat, to claim that it walked over the keyboard..